### DISCUSSION TITLE
Problems with upgrade to v5, less migration

### DISCUSSION THREAD

**Discussion by dazaart** [ant-design/ant-design#41616] [General]
Im following the upgrade guide of antd5 and have a problem with the point on [less migration](https://ant.design/docs/react/migration-v5#less-migration). I copied this to webpack config, then installted atn-design https://github.com/ant-design/ant-design/issues/20833. Now im getting following error:
![Screenshot (110)](https://user-images.githubusercontent.com/44365122/229570517-059680cb-40d2-45d1-8145-df739ec87721.png)

**Comment by WouterrV** [comment]
```
options : {
    options: 
```
looks funky, are you sure you need to nest options within another options?
